frontend/packages/gitops-plugin/locales/en/gitops-plugin.json

```diff
@@ -1,5 +1,7 @@
 {
-  "{{id}} by {{author}}": "{{id}} by {{author}}",
+  "{{msg}}": "{{msg}}",
+  "{{id}}": "{{id}}",
```

Comment:
I don't see a reason for this two attributes. Just skip i18n method in this case.

Reply:
Removed this in the latest update.